src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import AsyncView from '../../asyncView';
+import ApiForm from '../components/forms/apiForm';
+
+import Select2Field from '../components/forms/select2Field';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+
+import ProjectsStore from '../../../stores/projectsStore';
+
+import LoadingIndicator from '../../../components/loadingIndicator';
+
+const ACCOUNT_NOTIFICATION_FIELDS = {
+  'project-alerts/': {
+    name: 'subscribeByDefault',
+    type: 'select',
+    choices: [['default', 'Default'], ['on', 'On'], ['off', 'Off']],
+  },
+};
+
+export default class AccountNotificationDetails extends AsyncView {
+  getEndpoints() {
+    return [['notifications', '/users/me/notifications/'], ['projects', '/projects/']];
+  }
+
+  getFieldData(fieldName, projectList) {
+    ProjectsStore.loadInitialData(projectList);
+
+    const projectsByOrg = ProjectsStore.getAllGroupedByOrganization();
+
+    const fieldConfig = ACCOUNT_NOTIFICATION_FIELDS[fieldName];
+
+    // Display as select box in this view regardless of the type specified in the config
+    return Object.values(projectsByOrg).map(org => {
+      return {
+        name: org.organization.name,
+        projects: org.projects.map(project => {
+          return {
+            ...fieldConfig,
+            name: project.slug,
+            label: project.name,
+          };
+        }),
+      };
+    });
+  }
+
+  renderBody() {
+    if (this.state.loading) {
+      return <LoadingIndicator />;
+    }
+
+    const {path} = this.props.route;
+    const data = this.getFieldData(path, this.state.projects);
+
+    return (
+      <div>
+        <ApiForm apiMethod="PUT" apiEndpoint={'/users/me/notifications/'}>
+          <Panel>
+            <PanelHeader>
+              <div className="text-light">{this.props.route.name}</div>
```

Comment:
Can we use a styled component here?

Reply:
Yep. I'm gonna merge as-is for now though, and ask @Chrissy for help on the final styling of these components